### PR TITLE
Update ZapAddOn.xml

### PR DESCRIPTION
**Pull request by shekyan**
Update description of the add-on for better discoverability (folks who type `csp` should see this add-on)

### CODE REVIEW

File: src/org/zaproxy/zap/extension/cspscanner/ZapAddOn.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	<name>Content Security Policy Scanner</name>
 	<version>1</version>
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/cspscanner/ZapAddOn.xml]: Please, update the version to 2.

Comment (thc202) [inline src/org/zaproxy/zap/extension/cspscanner/ZapAddOn.xml]: Worth updating the changes element too.

**Comment by kingthorin** [comment]
Thanks @shekyan 

Please do an interactive rebase and fix-up or squash into a single "atomic" commit: https://github.com/zaproxy/zaproxy/blob/develop/CONTRIBUTING.md#what-we-zap-team-expect-from-you .

- https://www.kernel.org/pub/software/scm/git/docs/git-rebase.html#_interactive_mode
- https://wiki.eclipse.org/EGit/User_Guide#Interactive_Rebase

**Comment by shekyan** [comment]
@thc202 Addressed comments

**Comment by thc202** [comment]
Thanks!

**Comment by thc202** [comment]
The new version is now available in the marketplace.

**Pull request merged**